src/devtoolsPanel.ts

```diff
@@ -302,6 +304,44 @@ export class DevToolsPanel {
             return;
         }
 
+        const uri = await this.parseUrlToUri(url);
+
+        // Finally open the document if it exists
+        if (uri) {
+            const doc = await vscode.workspace.openTextDocument(uri);
+            void vscode.window.showTextDocument(
+                doc,
+                {
+                    preserveFocus: true,
+                    selection: new vscode.Range(line, column, line, column),
+                    viewColumn: vscode.ViewColumn.One,
+                });
+        } else {
+            void vscode.window.showErrorMessage(`Could not open document. No workspace mapping was found for '${url}'.`);
+        }
+    }
+
+    private async onSocketCssMirrorContent(message: string) {
+        if (!vscode.workspace.getConfiguration(SETTINGS_STORE_NAME).get('mirrorEdits')) {
+            return;
+        }
+
+        // Parse message and open the requested file
+        const { url, newContent } = JSON.parse(message) as ICssMirrorContentData;
+
+        const uri = await this.parseUrlToUri(url);
+
+        // Finally open and edit the document if it exists
+        if (uri) {
+            const enc = new TextEncoder();
+            const encodedNewContent = enc.encode(newContent);
+            void vscode.workspace.fs.writeFile(uri, encodedNewContent);
```

Comment:
Is there an option to replace the contents in the editor without writing the file to disk? Testing this with the live preview extension, I can't complete any CSS edits because when the file saves after my first keystroke the page reloads.